### ISSUE TITLE
Setup issue

### ISSUE THREAD

**Issue by lucode** [OCA/vertical-community#9]
Hi Yannick,
Hi everybody,

I did try to make a payment with the VC wallet. But an error occured. Either the error message says File! or currency!.
Nothing more. I could figure out that the central bank accounts in the ledger have the wrong currency, it was the same currency as the other accounts for bookkeeping, CHF. To switch the currencies to COM I had to activate the multi currency mode. Now all the wallet related accounts have the COM currency. But I still can't perform a transaction.
Any experience or suggestions?
I still not see through all the relations between ledger accounts and wallet.

Lucas

**Comment by YannickB** [comment]
Hello lucode,

Can you please provide screenshots of your process and configuration? I'd especially like to see the wallet configuration in Association / configuration / community

**Comment by lucode** [comment]
Hi Yannick,
I did set a fresh installation of an odoo DB and installed Swiss Localisation incl. Swiss Ledger. Then I did install markteplace. Everything looks ok, but I still get the error messages:
1. When I click on save.
![screenshot 2014-11-24_wallet-error](https://cloud.githubusercontent.com/assets/2928740/5166921/60ffae34-73f4-11e4-8a63-106eb367e98a.png)
2. When I try to chance the field unit price.
![screenshot 2014-11-24_wallet-error2](https://cloud.githubusercontent.com/assets/2928740/5166930/83776d6c-73f4-11e4-9c52-7965e0dd8bf2.png)

**Comment by YannickB** [comment]
Yes indeed we had some problem like this one last week, I suggest you update your code it shall be ok now.

**Comment by lucode** [comment]
Hi Yannick,
I did update the modules as you proposed but the errror is still there. But I figured out now a workaround.
When I delete the prexisting line (Unit Price, Currency, Subototal) and Add an new line (item) everything works fine. I think the error message is caused by a missing database record somewhere, that is behind the preconfigured line with 1 COM Unit price.

**Comment by YannickB** [comment]
Florent will check but I really think the bug is resolved. If you still have it after an update please provide some screenshots (unless the ones you provided are still valid?)

**Comment by YannickB** [comment]
For the record, this is the commit where I made the correction : 
https://github.com/OCA/vertical-community/commit/3fabcdab237a13a8ee411e84657cb7d3675a366b

**Comment by florent-valeureux** [comment]
Hi there, yes within French Ledger the bug was solved. I checked again this morning.
From my point of view you need to update again. 
http://i.imgur.com/WwemckD.png

**Comment by lucode** [comment]
I just pulled the actual files to my testsite, but the error still occurs. The screen shots are still valid the workaround too. Do I have to code the module for the Swiss Ledger first?
